src/test/java/org/ays/emergency_application/service/impl/EmergencyEvacuationApplicationServiceImplTest.java

```diff
@@ -23,6 +38,72 @@ class EmergencyEvacuationApplicationServiceImplTest extends AbstractUnitTest {
 
 
     private final EmergencyEvacuationApplicationRequestToEntityMapper emergencyEvacuationApplicationRequestToEntityMapper = EmergencyEvacuationApplicationRequestToEntityMapper.initialize();
+    private final EmergencyEvacuationApplicationEntityToEmergencyEvacuationApplicationMapper entityToEmergencyEvacuationApplicationMapper = EmergencyEvacuationApplicationEntityToEmergencyEvacuationApplicationMapper.initialize();
+
+    @Test
+    @SuppressWarnings("unchecked")
```

Comment:
Test kısmında neden SuppressWarnings kullanıldığını tam olarak anlayamadım.

Reply:
`Spesification` ve `Pageable` Interface'lerini doğrudan Mockito içerisinde kullandığımız için unchecked hatası veriyordu. Fakat burası test sınıfları olduğu için bu uyarının görünmesini istemiyorduk. O yüzden bunu ekleyerek o uyarının görünmemesini sağladık. Uygulamda kullanılan sınıflar olsaydı bu tarz bir anotasyonu eklemiyor olmamız gerekirdi.